src/Makefile

```diff
@@ -2852,302 +2852,302 @@ auto/gui_gtk_gresources.h: gui_gtk_res.xml $(GUI_GTK_RES_INPUTS)
 # specified for each file separately.
 
 objects:
-	mkdir objects
+	mkdir -p objects
 
-objects/arabic.o: arabic.c
+objects/arabic.o: objects arabic.c
```

Comment:
Ah, interesting.  I like that better than having the `mkdir` in each of the object file targets.  I just pushed another commit that implements your suggestion.  Thanks!